packages/next-swc/crates/next-core/js/src/overlay/internal/helpers/magic-identifier.ts

```diff
@@ -18,90 +18,91 @@ const enum Mode {
   LongHex,
 }
 
-const DECODE_REGEX = /^__TURBOPACK__([a-zA-Z0-9_$]+)__$/;
+const DECODE_REGEX = /^__TURBOPACK__([a-zA-Z0-9_$]+)__$/
 
 function decodeMagicIdentifier(identifier: string): string {
```

Comment:
There should be a comment linking back to the implementation in Rust code (and vice-versa). Also, why isn't this in turbopack itself?

Reply:
The error overlay will move back to turbopack, so it will be eventually there...

Reply:
Add a comment that points to the Rust file (and do the reverse on the other PR).